meilisearch-release-changelog/v1.2.0.md

```diff
@@ -0,0 +1,206 @@
+
+Meilisearch v1.2.0 improve search relevancy and performance but also introduces new filter operators like `IS NULL` and `IS EMPTY`!
+
+<!-- The following lines should ONLY be put in the official release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
+
+Some SDKs might not include all new features—consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+<!-- The following line should NOT be put in the official release changelogs, and the link should be added as soon as possible during the pre-release -->
+📖 Meilisearch's official documentation will be ready for the official v1.2.0 release. Meanwhile, work-in-progress documentation is available [here]().
+
+# New features and improvements 🔥
+
+## Delete documents by filter
+
+Expose a new route to delete documents by filters. As always, the fields you're filtering on must be set as [filterable](https://www.meilisearch.com/docs/reference/api/settings#update-filterable-attributes).
+
+The new route is located under `/indexes/:uid/documents/delete` and must be called with an HTTP `POST` request. Here is an example of usage using curl:
+
+```bash
+curl -X POST http://localhost:7700/indexes/dogs/documents/delete \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "filter": "doggo = golden" }'
+```
+
+As always, this returns a sum up of the task:
+
+```json
+{
+  "taskUid": 242,
+  "indexUid": "dogs",
+  "status": "enqueued",
+  "type": "documentDeletion",
+  "enqueuedAt": "2023-05-03T11:01:58.721841Z"
+}
+```
+
+Which you can then query in the task queue using the [task route](https://www.meilisearch.com/docs/reference/api/tasks#get-one-task).
+
+```json
+{
+  "uid": 242,
+  "indexUid": "dogs",
+  "status": "succeeded",
+  "type": "documentDeletion",
+  "canceledBy": null,
+  "details": {
+    "providedIds": 0,
+    "deletedDocuments": 2,
+    "originalFilter": "\"doggo = golden\""
+  },
+  "error": null,
+  "duration": "PT0.005040S",
+  "enqueuedAt": "2023-05-03T11:01:58.721841Z",
+  "startedAt": "2023-05-03T11:01:58.722442Z",
+  "finishedAt": "2023-05-03T11:01:58.727482Z"
+}
+```
+
+Done by @dureuill and @irevoire in #3550.
+
+## Get documents by filter
+
+To help you with the document deletion by filters, we also decided to let you get documents by filters. This means the `GET` `/indexes/:uid/documents` route now also supports receiving a filter.
```

Comment:
@macraig how can we use it? an example of usage would be perfect.

Reply:
I pushed an example with curl